packages/react/react/src/hooks/useImageLoad.ts

```diff
@@ -0,0 +1,46 @@
+import { noop } from '@toss/utils';
+import { useCallback, useState } from 'react';
+import { usePreservedCallback } from './usePreservedCallback';
+
+interface Options {
+  onLoadComplete?: () => void;
+  onLoadStart?: () => void;
```

Comment:
It's very minor, but it would be nice to arrange it according to the order in which it is used!

```suggestion
  onLoadStart?: () => void;
  onLoadComplete?: () => void;
```